src/Microsoft.SqlTools.ServiceLayer/Security/LoginData.cs

```diff
@@ -864,6 +864,10 @@ private void PopulateServerRolesForAzure()
     internal class LoginPrototype
     {
         private SqlCollationSensitiveStringComparer comparer = null;
+        private static string AZURE_SERVER_ROLE_ALTER_QUERY =
```

Comment:
no SMO support for this?

Reply:
no SMO support for now, I'll need to make a SMO fix to support server role related operations for Azure SQL DB